operations/wallet-service/modules/database/db_functions.bal

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+//
+// This software is the property of WSO2 LLC. and its suppliers, if any.
+// Dissemination of any information or reproduction of any material contained
+// herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+// You may not alter or remove any copyright or other notice from copies of this content.
+import wallet_service.types;
+
+import ballerina/log;
+import ballerina/sql;
+
+# Check if wallet already exists or not.
+#
+# + walletAddress - Wallet address
+# + return - True if wallet exists, false if not exists, error if error occurred
+public isolated function isUserWalletExists(string walletAddress) returns boolean|sql:Error {
+    types:UserWallet|sql:Error walletResponse = dbClient->queryRow(getUserWalletQuery(walletAddress));
+    if walletResponse is sql:NoRowsError {
+        return false;
+    } else if walletResponse is sql:Error {
+        log:printError("Error while checking user wallet", walletResponse, info = walletResponse.toString());
+        return walletResponse;
+    }
+    return true;
```

Comment:
```suggestion
    if walletResponse is sql:Error && walletResponse !is sql:NoRowsError {
        log:printError("Error while checking user wallet", walletResponse, info = walletResponse.toString());
        return walletResponse;
    }
    
    return walletResponse is types:UserWallet;
```
I feel that the above looks better. Test this before implementing.

Reply:
[8c5ce54](https://github.com/wso2/digiops-currency/pull/22/commits/8c5ce54e2093f7e170daf87793924ef8b54612a4)